test/Tizen.NUI.Tests/Tizen.NUI.Devel.Tests/testcase/public/Images/TSNativeImageSource.cs

```diff
@@ -88,6 +88,7 @@ public void NativeImageSourceAcquireBuffer()
             try
             {
                 testingTarget.AcquireBuffer(ref width, ref height, ref stride);
+                testingTarget.ReleaseBuffer(); ///< Must call after AcquireBuffer
```

Comment:
This is a minor opinion.

If ReleaseBuffer is needed, in this TC it seems better to place ReleaseBuffer before Dispose() outside of try~catch.

try
{
  testingTarget.AcquireBuffer(ref width, ref height, ref stride);
} catch (Exception e) { ... }

testingTarget.ReleaseBuffer(); ///< Must call after AcquireBuffer
testingTarget.Dispose();

Reply:
If AcquireBuffer succeeds and ReleaseBuffer fails, problems may arise. It would be better to keep it inside a try-catch like now. 

Please ignore my previous comment

Reply:
I though same thing. But also, if their was some exception occured, internal mutex "might" not be locked. In this case, we should not call ReleaseBuffer();

So I make it inside of try.